app/services/check_new_matches_service.rb

```diff
@@ -0,0 +1,31 @@
+class CheckNewMatchesService
+  def initialize(params)
+    @new_target = params[:new_target]
+    @current_user = params[:current_user]
+  end
+
+  def check_new_matches()
+    new_target_coord = Geokit::LatLng.new(@new_target.latitude,@new_target.longitude)
+
+    new_matches = UserTarget.within(@new_target.area, origin: new_target_coord).not_from_user(@new_target.user_id).with_topic(@new_target.topic_id)
```

Comment:
Try to make your lines shorter, you can break your lines:
```
new_matches = UserTarget.within(@new_target.area, origin: new_target_coord)
              .not_from_user(@new_target.user_id).with_topic(@new_target.topic_id)
```

https://github.com/rootstrap/guides/tree/master/ruby#consistent-multi-line-chains